static/logo.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg id="a03ed06d-885d-4bed-8a4a-44974750f5a8" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 501.34 185.17"><defs><style>.a7dc26de-4b77-4d05-9982-f373222849ac,.be9c742b-a6b1-41a4-97ac-f5e00be0ef3e{fill:#ee362d;}.ab3dba79-30e8-459a-a0be-ce4217101439{fill:#6d6f6e;}.a7dc26de-4b77-4d05-9982-f373222849ac{font-size:54.24px;font-family:Montserrat-Medium, Montserrat;font-weight:500;}.abea20a3-df56-4a97-97f6-6eaf4c409d82,.fe344c9d-5a72-4e1f-9d1e-725524f3af52{letter-spacing:0.01em;}.b5a0f27e-7f61-4ece-80f2-d512df9640a8,.e2050f92-abb6-4b6b-a25b-419347361e9c,.e32e9238-6004-409e-a320-45826ebea3e8,.f47252b2-8628-4792-bd7e-e4346d71e80c{font-size:50.01px;}.b5a0f27e-7f61-4ece-80f2-d512df9640a8,.e2050f92-abb6-4b6b-a25b-419347361e9c{letter-spacing:0.01em;}.abea20a3-df56-4a97-97f6-6eaf4c409d82,.b5a0f27e-7f61-4ece-80f2-d512df9640a8,.e32e9238-6004-409e-a320-45826ebea3e8,.f47252b2-8628-4792-bd7e-e4346d71e80c{fill:#6d6e70;}.f47252b2-8628-4792-bd7e-e4346d71e80c{letter-spacing:0em;}</style></defs><rect class="be9c742b-a6b1-41a4-97ac-f5e00be0ef3e" x="70.09" y="68.78" width="38.79" height="10.38" rx="1.94"/><rect class="be9c742b-a6b1-41a4-97ac-f5e00be0ef3e" x="70.09" y="87.47" width="24.59" height="10.38" rx="1.94"/><rect class="ab3dba79-30e8-459a-a0be-ce4217101439" x="104.96" y="87.47" width="24.59" height="10.38" rx="1.94"/><rect class="be9c742b-a6b1-41a4-97ac-f5e00be0ef3e" x="70.09" y="106.01" width="10.38" height="10.38" rx="1.94"/><rect class="ab3dba79-30e8-459a-a0be-ce4217101439" x="119.17" y="68.78" width="10.38" height="10.38" rx="1.94"/><rect class="ab3dba79-30e8-459a-a0be-ce4217101439" x="90.76" y="106.01" width="38.79" height="10.38" rx="1.94"/><text class="a7dc26de-4b77-4d05-9982-f373222849ac" transform="translate(142.91 113.34)"><tspan class="fe344c9d-5a72-4e1f-9d1e-725524f3af52">F</tspan><tspan class="e2050f92-abb6-4b6b-a25b-419347361e9c" x="34.98" y="0">ORGE</tspan><tspan class="abea20a3-df56-4a97-97f6-6eaf4c409d82" x="165" y="0">F</tspan><tspan class="f47252b2-8628-4792-bd7e-e4346d71e80c" x="199.98" y="0">L</tspan><tspan class="e32e9238-6004-409e-a320-45826ebea3e8" x="225.24" y="0">U</tspan><tspan class="b5a0f27e-7f61-4ece-80f2-d512df9640a8" x="259.24" y="0">X</tspan></text></svg>
```

Comment:
The logo styling changes in different zoom levels, can you contact Sai Pranay and get this fixed?

https://user-images.githubusercontent.com/33521410/150632830-311dd2db-64d0-4a08-9420-ca9fc459594b.mp4

Reply:
Interesting. Sure, I'll get in touch with him.

Reply:
The issue has been resolved.